plugins/modules/user.py

```diff
@@ -141,6 +142,9 @@ def modify_user(module):
                 opts += "%s=%s " % (attr, val)
         if load_module_opts is not None:
             opts = load_module_opts + opts
+        pattern = re.compile(r'(true|yes|always|false|no|never)', re.IGNORECASE)
```

Comment:
@JLRebello 
does this work for the case where user tries to modify
`account_locked` and `capabilities` at the same time.
say `account_locked` is `True` and `capanilities` is `CAP_AACCT`
i assume you would make opts lower case at this point? which also
incidentally converts `CAP_AACCT` to lower case, which would be an
invalid input for `capabilities`

you suggestion is
```
for attr, val in attributes.items():
    if attr == 'load_module':
        load_module_opts "-R %s " % val
    else:
        if val in ["True, False, Yes, No, Always, Never"]:
            val = val.lower()
        opts += "%s=%s " % (attr, val)
```